src/pages/workspace/accounting/xero/export/XeroBankAccountSelectPage.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useCallback, useMemo} from 'react';
+import {View} from 'react-native';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import type {SelectorType} from '@components/SelectionScreen';
+import SelectionScreen from '@components/SelectionScreen';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Connections from '@libs/actions/connections';
+import Navigation from '@libs/Navigation/Navigation';
+import type {WithPolicyConnectionsProps} from '@pages/workspace/withPolicyConnections';
+import withPolicyConnections from '@pages/workspace/withPolicyConnections';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function XeroBankAccountSelectPage({policy}: WithPolicyConnectionsProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const policyID = policy?.id ?? '';
+    const {bankAccounts} = policy?.connections?.xero?.data ?? {};
+
+    const {nonReimbursableAccount} = policy?.connections?.xero?.config.export ?? {};
+
+    const xeroSelectorOptions = useMemo<SelectorType[]>(
+        () =>
+            (bankAccounts ?? []).map(({id, name}) => ({
+                value: id,
+                text: name,
+                keyForList: id,
+                isSelected: nonReimbursableAccount === id,
```

Comment:
nonReimbursableAccount is an object. This comparison is false?

Reply:
It's a string?

https://github.com/Expensify/App/blob/a5bd1f2f7941fd65a5cbf647ac64691cd51b7bc6/src/types/onyx/Policy.ts#L252

Reply:
Ah okay. can we rename it to nonReimbursableAccountID then?

Reply:
No we can't as it's the property returned from IS 😅

<img width="503" alt="image" src="https://github.com/Expensify/App/assets/21549771/2b570416-b98b-4e11-aba5-077bc124816d">

See [connection's data example](https://docs.google.com/document/d/1VEIzxwNzedLNestAFVT5InGvD4N6FHb0922xjMGOkko/edit#bookmark=id.c7yrbwrwgmlf) from the Xero DD.